public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx

```diff
@@ -139,21 +162,30 @@ export class PromQueryEditor extends PureComponent<PromQueryEditorProps, State>
 
             <div className="gf-form">
               <InlineFormLabel
-                width={7}
+                width={5}
                 tooltip={
                   <>
-                    An additional lower limit for the step parameter of the Prometheus query and for the{' '}
-                    <code>$__interval</code> and <code>$__rate_interval</code> variables. The limit is absolute and not
-                    modified by the &quot;Resolution&quot; setting.
+                    An additional limit for the step parameter of the Prometheus query and for the{' '}
```

Comment:
I think this whole thing needs rewriting, what do you think of the following:
```
Optionally set lower or upper bounds on the interval between datapoints, e.g., set "minimum 1h" to hint that the measurements were not taken more frequently. <code>$__interval</code> and <code>$__rate_interval</code> are supported. 
 ```
I would ignore the "exact" use case, since it's self-explanatory.

Reply:
Yeah i think that sounds better!